ECMA2Yaml/ECMAHelper/Models/VersionedValue.cs

```diff
@@ -17,6 +18,23 @@ public VersionedString(HashSet<string> monikers, string value) : base(monikers,
         }
     }
 
+    public class VersionedReturnType : VersionedString
+    {
+        public string RefType { get; set; }
+        public VersionedReturnType() { }
+        public VersionedReturnType(HashSet<string> monikers, string value, string reftype) : base(monikers, value)
+        {
+            RefType = reftype;
+        }
+    }
+
+    public class VersionedTypeReference : VersionedValue<TypeReference>
```

Comment:
is `VersionedTypeReference` used?

Reply:
Yes, but this question made me realize that I wasn't fully propagating the value to the end yaml ... it is now :) So this adds "refType" to the yaml, which is not something that we're rendering today, but we will have to at some point since we have `ref returns` in C#. I didn't want to mess with the single return output, so I didn't change that data type to include it (it's using TypeReference right now); but this at least gives us an easier line of evolution going forward once we decide to implement ref returns.